app/components/ak-svg/am-loading.hbs

```diff
@@ -0,0 +1,167 @@
+<svg
```

Comment:
This svg is not specific to app monitoring so can we rename this to something like `empty-loading-list.hbs`